src/main/java/org/isf/medicalinventory/gui/InventoryEdit.java

```diff
@@ -865,8 +874,44 @@ private JButton getCloseButton() {
 		});
 		return closeButton;
 	}
-	
-	private JButton getCleanTableButton() {
+
+	private JButton getPrintButton() {
+		printButton = new JButton(MessageBundle.getMessage("angal.common.print.btn"));
+		printButton.setMnemonic(MessageBundle.getMnemonic("angal.common.print.btn.key"));
+		printButton.setEnabled(true);
+
+		printButton.addActionListener(e -> {
+			if (inventory == null || inventory.getId() <= 0) {
+				MessageDialog.info(this, "angal.inventory.pleasesavebeforprinting");
+				return;
+			}
+
+			if (InventoryStatus.done.name().equals(inventory.getStatus())) {
+				int printQtyReal = 0;
+				int response = MessageDialog.yesNo(this, "angal.inventorywardedit.askforrealquantityempty");
+				if (response == JOptionPane.YES_OPTION) {
+					printQtyReal = 1;
+				}
+
+				try {
+					JasperReportResultDto reportResult = jasperReportsManager.getInventoryReportPdf(inventory, "Inventory", printQtyReal);
+					if (reportResult != null) {
+						MessageDialog.info(this, "angal.inventory.print.success.msg");
+					} else {
+						MessageDialog.error(this, "angal.inventory.printing.error.msg");
+					}
```

Comment:
I don't get this: the report should be shown to the user before printing (not only created as PDF) and in case the report is OK the user can decide to print or not from the interface (see INTERNALVIEWER and other reports in general).

There's no need of a print.success.msg.